uitools/common/src/NetworkAuthenticationChallengeRelay.h

```diff
@@ -16,26 +16,26 @@
 #ifndef ESRI_ARCGISRUNTIME_TOOLKIT_NETWORKAUTHENTICATIONCHALLENGERELAY_H
 #define ESRI_ARCGISRUNTIME_TOOLKIT_NETWORKAUTHENTICATIONCHALLENGERELAY_H
 
-// ArcGIS Maps SDK headers
+// STL headers
```

Comment:
Another one. I won't comment on these after this one. I assume there will be a handful scattered around.